### DISCUSSION TITLE
Någon som vet?

### DISCUSSION THREAD

**Discussion by Lunarwalker60** [stefansundin/privatkopiera#204] [Q&A]
Hur får man ffmpeg att ladda ner avkrypterat med ett manifest som detta:

#EXTM3U
#EXT-X-VERSION:6
#
# Created with Unified Streaming Platform (version=1.11.23-28141)
#EXT-X-SESSION-KEY:METHOD=SAMPLE-AES,KEYID=0xde841f64889330f88c58794a561d726c,URI="data:text/plain;base64,AAAAYnBzc2gAAAAA7e+LqXnWSs6jyCfc1R0h7QAAAEISEN6EH2SIkzD4jFh5SlYdcmwiKGV5SmhjM05sZEVsa0lqb2lkSFp0WldScFlTMHlNRGN6TURrek1TSjlI88aJmwY=",KEYFORMAT="urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed",KEYFORMATVERSIONS="1"

#EXT-X-SESSION-KEY:METHOD=SAMPLE-AES,URI="data:text/plain;charset=UTF-16;base64,MgMAAAEAAQAoAzwAVwBSAE0ASABFAEEARABFAFIAIAB4AG0AbABuAHMAPQAiAGgAdAB0AHAAOgAvAC8AcwBjAGgAZQBtAGEAcwAuAG0AaQBjAHIAbwBzAG8AZgB0AC4AYwBvAG0ALwBEAFIATQAvADIAMAAwADcALwAwADMALwBQAGwAYQB5AFIAZQBhAGQAeQBIAGUAYQBkAGUAcgAiACAAdgBlAHIAcwBpAG8AbgA9ACIANAAuADMALgAwAC4AMAAiAD4APABEAEEAVABBAD4APABQAFIATwBUAEUAQwBUAEkATgBGAE8APgA8AEsASQBEAFMAPgA8AEsASQBEACAAQQBMAEcASQBEAD0AIgBBAEUAUwBDAEIAQwAiACAAVgBBAEwAVQBFAD0AIgBaAEIAKwBFADMAcABPAEkAKwBEAEMATQBXAEgAbABLAFYAaAAxAHkAYgBBAD0APQAiAD4APAAvAEsASQBEAD4APAAvAEsASQBEAFMAPgA8AC8AUABSAE8AVABFAEMAVABJAE4ARgBPAD4APABMAEEAXwBVAFIATAA+AGgAdAB0AHAAcwA6AC8ALwBsAGkAYwAuAGQAcgBtAHQAbwBkAGEAeQAuAGMAbwBtAC8AbABpAGMAZQBuAHMAZQAtAHAAcgBvAHgAeQAtAGgAZQBhAGQAZQByAGEAdQB0AGgALwBkAHIAbQB0AG8AZABhAHkALwBSAGkAZwBoAHQAcwBNAGEAbgBhAGcAZQByAC4AYQBzAG0AeAA8AC8ATABBAF8AVQBSAEwAPgA8AEwAVQBJAF8AVQBSAEwAPgBoAHQAdABwAHMAOgAvAC8AcABsAGEAeQByAGUAYQBkAHkALQB1AGkALgBlAHgAYQBtAHAAbABlAC4AYwBvAG0APAAvAEwAVQBJAF8AVQBSAEwAPgA8AEQARQBDAFIAWQBQAFQATwBSAFMARQBUAFUAUAA+AE8ATgBEAEUATQBBAE4ARAA8AC8ARABFAEMAUgBZAFAAVABPAFIAUwBFAFQAVQBQAD4APAAvAEQAQQBUAEEAPgA8AC8AVwBSAE0ASABFAEEARABFAFIAPgA=",KEYFORMAT="com.microsoft.playready",KEYFORMATVERSIONS="1"

#EXT-X-SESSION-KEY:METHOD=SAMPLE-AES,URI="skd://drmtoday?assetId=tvmedia-20730931&variantId&keyId=de841f64889330f88c58794a561d726c",KEYFORMAT="com.apple.streamingkeydelivery",KEYFORMATVERSIONS="1"

#EXT-X-SESSION-KEY:METHOD=SAMPLE-AES,KEYID=0xde841f64889330f88c58794a561d726c,URI="data:text/plain;base64,AAAAn3Bzc2gAAAAAPV5tNZuaQei4Q908bnLELAAAAH97InZlcnNpb24iOiJWMS4wIiwia2lkcyI6WyIzb1FmWklpVE1QaU1XSGxLVmgxeWJBPT0iXSwiY29udGVudElEIjoiZXlKaGMzTmxkRWxrSWpvaWRIWnRaV1JwWVMweU1EY3pNRGt6TVNKOSIsImVuc2NoZW1hIjoiY2JjcyJ9",KEYFORMAT="urn:uuid:3d5e6d35-9b9a-41e8-b843-dd3c6e72c42c",KEYFORMATVERSIONS="1"

# AUDIO groups
#EXT-X-MEDIA:TYPE=AUDIO,GROUP-ID="audio-aacl-128",NAME="audio",DEFAULT=YES,AUTOSELECT=YES,CHANNELS="2",URI="680c0240-8394-11ee-85d1-73eb62738ad6_20730931-audio=128000.m3u8"

# SUBTITLES groups
#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="textstream",LANGUAGE="sv",NAME="Svenska",DEFAULT=YES,AUTOSELECT=YES,URI="680c0240-8394-11ee-85d1-73eb62738ad6_20730931-textstream_swe=1000.m3u8"

# variants
#EXT-X-STREAM-INF:BANDWIDTH=985000,CODECS="mp4a.40.2,avc1.4D401E",RESOLUTION=640x360,FRAME-RATE=25,AUDIO="audio-aacl-128",SUBTITLES="textstream",CLOSED-CAPTIONS=NONE
680c0240-8394-11ee-85d1-73eb62738ad6_20730931-video=800000.m3u8

**Comment by karlingen** [comment]
Lyckades du hitta nån lösning till det här?

**Comment by Lunarwalker60** [comment]
Nej, jag gav upp det ganska snabbt.--Sent from my Android phone with mail.com Mail. Please excuse my brevity.On 2024-10-07, 22:28 Karl Metum ***@***.***> wrote:

   
  Lyckades du hitta nån lösning till det här?
